src/pages/CatInfoPage.js

```diff
@@ -0,0 +1,135 @@
+import axios from 'axios';
+import styled from 'styled-components';
+import { useNavigate } from 'react-router-dom';
+import { useState, useEffect } from 'react';
+
+import coralcatleft from '../images/coral_cat_head_left.svg';
+
+export default function CatInfoPage() {
+  const navigate = useNavigate();
+  const [catData, setCatData] = useState([]);
+  const apiKey = process.env.API_KEY;
+
+  useEffect(
+    () => {
+      axios
+        .get('https://api.thecatapi.com/v1/breeds', {
+          headers: {
+            'x-api-key': apiKey,
+          },
+        })
+        .then(response => {
+          return response.data;
+        })
+        .then(data => {
+          console.log(data);
+          setCatData(data);
+        })
+        .catch(error => console.log(error, 'sorry, there is a problem'));
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
dont know if you wanna take it out as it is just a comment?😊

Reply:
unfortunately not so easy done, it is because of the empty array dependency of the UseEffect for the single fetch